### ISSUE TITLE
Download associated files

### ISSUE THREAD

**Issue by TomWhitson** [dialect-katrineholm/laravel-gdpr-compliance#77]
Is there a good way to extend this to download all user files (either along with associated models or as a zipped bundle)

**Comment by dsbilling** [comment]
Yes, with this: [https://github.com/kfvit/laravel-gdpr-compliance#lazy-eager-loading-relationships](https://github.com/kfvit/laravel-gdpr-compliance#lazy-eager-loading-relationships)

**Comment by TomWhitson** [comment]
Perhaps I was not clear, I've got file paths stored in the model as text so obviously in the download this just comes through as text. I was looking to see if there's a way to download these as files.

**Comment by dsbilling** [comment]
Ah, I see now. I would build upon this package to get what you want. I think this is a good idea to implement. If you find a good solution make a PR.
